docs/features/specification.md

```diff
@@ -79,7 +79,18 @@ spec:
       # Reference to service that the rollout modifies as the active service.
       # Required.
       activeService: active-service
-
+      previewService: bluegreen-preview
```

Comment:
I think there is already a previewService in this file. Could you delete that one?